DEVELOPER.md

```diff
@@ -44,7 +44,13 @@ and follow the wizard.
 
 ### How to run e2e Tests
 
-Run:
-```shell
-make e2e
-```
+End-to-end (e2e) tests rely on a `getenvoy` binary that defaults to what was built by `make bin`.
+
+In simplest case, execute `make e2e` to run all tests configured. You can also set the below ENV variables to effect the
```

Comment:
this makes me think that `make e2e` run all tests for "all languages". So maybe it would be helpful if we add `make e2e all` or similar for running like that?

Reply:
maybe e2e.all as that wouldn't effect args?

Reply:
sg